meta-resin-common/conf/distro/include/resin-os.inc

```diff
@@ -74,3 +74,4 @@ RESIN_FLASHER_FLAG_FILE = "resin-image-flasher"
 INITRAMFS_IMAGE = "resin-image-initramfs"
 INITRAMFS_IMAGE_BUNDLE = "1"
 KERNEL_INITRAMFS = "-initramfs"
+INITRAMFS_TASK = ""
```

Comment:
You only get the circular dependency if the two variables (i.e. INITRAMFS_IMAGE, INITRAMFS_TASK) are used.